### PR TITLE
Add more details to README.md

### PR DESCRIPTION
**Pull request by x389liu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,15 +11,97 @@ Currently, we use Wikidata as a stand-in knowledge base as the source of ground-
...
+pushd src/main/resources/solr && ./solr.sh ../../../../solrini localhost:9983 && popd
+```
+   
+Solr should now be available at [http://localhost:8983/](http://localhost:8983/) for browsing.
+
+### Indexing into SolrCloud from Anserini
+
+Index a document collection with Anserini, and ensure the appropriate Solr [command-line parameters](https://github.com/dstlry/dstlr/blob/master/src/main/scala/io/dstlr/package.scala) for `dstlr` are adjusted if use non-default options. 
+
+Note: Remember to always rebuild the JAR by running `sbt assembly` if any changes are made to [command-line parameters](https://github.com/dstlry/dstlr/blob/master/src/main/scala/io/dstlr/package.scala).
```
Comment (r-clancy) [inline README.md]: This is a bit out of place here?

File: README.md [hunk 2]
```diff
@@ -55,11 +137,59 @@ CREATE INDEX ON :Relation(type, confidence)
...
 
-Note that each script will need to be modified based on your environment (e.g., available memory, number of executors, Solr, etc.) - options available [here](src/main/scala/io/dstlr/package.scala).
+Note: Modify `extract.sh` based on your environment (e.g., available memory, number of executors, Solr, etc.) - options available [here](src/main/scala/io/dstlr/package.scala).
+
+After the extraction is done, check if an output folder (called `triples/` by default) is created, and several Parquet files are generated inside the output folder.
+
+In case you want to view the Parquet file in JSON format:
+
+```
+parquet-tools cat --json [filename]
```
Comment (r-clancy) [inline README.md]: Maybe info on how to download parquet-tools?

Comment (r-clancy) [inline README.md]: Maybe info on how to download parquet-tools?

Comment (x389liu) [reply README.md]: right, will do!

File: README.md (hunk 1 above)
Comment (x389liu) [reply README.md]: which part do you mean? the part mentioning we need to do a rebuild or the part about changing parameters?

Comment (r-clancy) [reply README.md]: Actually, this line should be removed I think. Command-line parameters should be modified in the scripts in the `bin` folder, not modifying the default values in the Scala code.

Comment (x389liu) [reply README.md]: yea makes sense. will remove it.

**Pull request merged**